RELEASE_PROCESS.md

```diff
@@ -0,0 +1,8 @@
+* Checkout master: `git checkout master`.
+* Get latest: `git pull --rebase origin master`
+* Bump version number in `lib/gnar-style/version.rb`.
+* Update the [CHANGELOG](CHANGELOG.md), moving the current release work to under the version number.
+* Create a commit, including the version number in the commit message: `git add
+  --all && git commit -m "Version x.y.z"`.
+* Push the release to RubyGems: `rake release`.
```

Comment:
Does `bundle exec` matter here?

Reply:
It hasn't come up as an issue for me. That doesn't mean it doesn't matter, but I haven't had a need to use it here.